courses/free-courses-ko.md

```diff
@@ -39,7 +41,7 @@
 * [Windows](#windows)
 
 
-### Algorithms & Data Structures
+### <a id="algorithms-&-data-structures"></a>Algorithms & Data Structures
```

Comment:
`&` is not allowed. The link in the index should be changed to `#algorithms--data-structures`

Reply:
Ok

Reply:
Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

Reply:
Resolved at #7029 with commit 0f17bd78a3fdedd83b651eee6200d6b5b2abd67b